web/tsconfig.json

```diff
@@ -17,14 +17,42 @@
     "module": "ES2022",
     "baseUrl": "./",
     "paths": {
-      "@repositories/*": ["src/repositories/*"],
-      "@models/*": ["src/models/*"],
-      "@mediators/*": ["src/mediators/*"],
-      "@utils/*": ["src/utils/*"],
-      "@components/*": ["src/app/components/*"],
-      "@features/*": ["src/app/features/*"],
-      "@app/*": ["src/app/*"],
-      "@triggo-ui/*": ["./libs/triggo-ui/*"]
+      "@repositories/*": [
+        "src/repositories/*"
+      ],
+      "@models/*": [
+        "src/models/*"
+      ],
+      "@mediators/*": [
+        "src/mediators/*"
+      ],
+      "@utils/*": [
+        "src/utils/*"
+      ],
+      "@components/*": [
+        "src/app/components/*"
+      ],
+      "@features/*": [
+        "src/app/features/*"
+      ],
+      "@app/*": [
+        "src/app/*"
+      ],
+      "@triggo-ui/*": [
+        "./libs/triggo-ui/*"
+      ],
+      "@spartan-ng/ui-input-helm": [
+        ".\\libs/triggo-ui/ui-input-helm/src/index.ts"
+      ],
+      "@spartan-ng/ui-formfield-helm": [
+        ".\\libs/triggo-ui/ui-formfield-helm/src/index.ts"
```

Comment:
These things are automatically added to the `tsconfig.json` by the spartan/ui cli, try to remove them as soon as you add a component